app/dashboard/user/features/RowCard.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @next/next/no-img-element */
+import clsx from "clsx";
+
+type RowCardProps = {
+  image: string;
+  label: string;
+  isActive?: boolean;
+  onClick?: () => void;
+};
+export const RowCard = (props: RowCardProps) => {
+  const { label, isActive, onClick, image } = props;
+  return (
+    <div
+      className={clsx(
+        "flex items-center h-[50px] rounded-r-full cursor-pointer transition-all duration-300 group",
+        isActive ? "bg-gray-300" : "hover:bg-gray-300"
+      )}
+      onClick={onClick}
+    >
+      <div
+        className={clsx(
+          "h-full w-[5px] rounded-lg",
+          isActive ? "bg-primary" : "group-hover:bg-primary"
+        )}
+      ></div>
+      <img src={image} alt="" className="ml-5 h-6 w-6 shrink-0 " />
```

Comment:
Please use the next JS image component instead. We're using the "sharp" package native to Next JS and it only works with the provided NextJS image component